static/src/stylesheets/module/atoms/_youtube.scss

```diff
@@ -268,4 +268,9 @@
         }
     }
 
+    .media-primary & {
+        @include mq($from: mobile, $until: mobileLandscape) {
+            margin-top: -1rem;
```

Comment:
BEM naming should do enough to denote parent/child relationships. I don't think nesting should do that.

As a guidance we only nest which we don't have another easy choice.